feedback-rep2004.md

```diff
@@ -0,0 +1,44 @@
+An example of this is a package not available on one of the Tier1 platforms, thus not qualifying to be quality level < 4. But being production ready and complying with every other criterium to be level 1.
+We think adding a full 'report card' could be useful to assess how a package is failing to meet criteria for the higher quality level. For this 'report card' to be useful, it should be quickly and easily readable. We could consider a 'spider chart' (or [radar chart as per wikipedia](https://en.wikipedia.org/wiki/Radar_chart)), or a table like [here](https://github.com/ros-infrastructure/rep/blob/c8bd456f531acc4863d0bb8888c28b10f9271492/rep-2004.rst#quality-level-comparison-chart).
+
+### Security specific context and feedback
+
+#### Quality Levels and the Vulnerability Disclosure Policy
+The VDP tells the public we’d like a flat 90-day grace period between when we receive a vulnerability before it’s made public.  In the interest of keeping the VDP simple, I recommend keeping code quality out of this public-facing document.  We can--and should--strive to do better for high quality code, but I don’t believe that needs to be a part of this document.
+
+#### Quality Levels and Vulnerability Remediation
+Internally, however, we should set a higher standard for higher quality code.
+Level 1 and 2 code will be widely deployed in the field. Fixes should be made available this code quickly to protect production robots.
+Level 3 and some level 4 code may not be deployed in robots, but will be used daily by firms that build robots.  High risk vulnerabilities should be handled quickly to protect development environments.
+CVSS is the defacto standard for risk assessing vulnerabilities.  Although flawed, it’s better than trying to create a new standard.
+
+#### Remediation Timeline
+Here’s an initial proposal on responsiveness based on the principles above.  This would be a commitment by the ROS 2 developer community.
+Triage within 7 natural days (one week) for all reports, triage assigns a CVSS score and a maintainer to provide a fix
+Maintainer is expected to provide a fix according to this timeline; the maintainer may also change the CVSS score as they dig into the issue:
+
+
+|                   | Critical/High Risk<br> CVSS 7.0+ | Medium Risk<br> CVSS 4.0-6.9            | Low Risk<br>CVSS <3.9                       |
+| ----------------- | --------- | ---------------------- | ------------------------------ |
+| Quality Level 1   | 7 days    | 30 days                | No timeline, use issue         |
+| Quality Level 2   | 7 days    | 30 days                | No timeline, use issue         |
+| Quality Level 3   | 30 days   | No timeline, use issue | No timeline, use issue         |
+| Quality Level 4-5 | 30 days   | No timeline, use issue | No timeline, use issue         |
```

Comment:
I doubt levels 4-5 would ever see a fix, and there's no lower category to fall to, which means specifying this doesn't really mean anything. Should we remove it?